test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -919,6 +919,16 @@ def forward(self, x, data):
         update = torch.randn(1, 2)
         self.run_test(CopyModel3(), (x, update))
 
+        class CopyModel4(torch.nn.Module):
```

Comment:
nit: we will have to come up with some conventions to uniformize the tests (when to have multiple run_test() under the same test function, do we want to add a separate error message in case the test fails to identify which call to run_test failed, etc.). 

(eg. Here, there are multiple classes CopyModel# defined under the same test_copy_(), but test_index_put() also defines the same model as CopuModel4(). When do we want to do something similar to CopyModel3(), and merge them by running 2 instances of run_test() on the same model with different inputs, vs. when do we want to keep them separate.)